modules/Cargo.toml

```diff
@@ -45,19 +45,23 @@ num-traits = { version = "0.2.15", default-features = false }
 derive_more = { version = "0.99.17", default-features = false, features = ["from", "display"] }
 
 [dependencies.tendermint]
-version = "=0.23.7"
+version = "=0.24.0-pre.2"
 default-features = false
 
 [dependencies.tendermint-proto]
-version = "=0.23.7"
+version = "=0.24.0-pre.2"
+default-features = false
+
+[dependencies.tendermint-rpc]
```

Comment:
Here's another gotcha: `ibc` cannot begin to depend on `tendermint-rpc` because it must be no_std compatible...